backend/src/controllers/userControllers.js

```diff
@@ -10,13 +13,47 @@ const browse = async (req, res, next) => {
     if (users == null) {
       res.sendStatus(404);
     } else {
+      users.forEach((e) => {
+        delete e.password;
+      });
       res.status(200).json(users);
     }
   } catch (error) {
     next(error);
   }
 };
 
+const download = async (req, res, next) => {
+  try {
+    const users = await tables.user.readAll();
+
+    if (users == null) {
+      res.sendStatus(404);
+    } else {
+      users.forEach((e) => {
+        delete e.password;
+      });
+      const test = [{ "sep=,": "" }];
```

Comment:
Please comment why you need to use this line (as in : it's necessary to make the CSV readable)

Reply:
check the changes and you can resolve this conversation @QuentinVerwaerde